ui/core/src/actions/wallet.ts

```diff
@@ -32,7 +33,16 @@ export default ({
   };
 
   effect(() => {
-    store.wallet.sif.isConnected = state.connected;
+    if (store.wallet.sif.isConnected !== state.connected) {
```

Comment:
I think effects here will be removed for Keplr

Reply:
We have to report connected status somehow. Have you refactored to an event handler?

Reply:
No, just async for now